docs/book/features/cloud-pipelines/guide-aws-gcp-azure.md

```diff
@@ -37,127 +29,122 @@ If you are doing a manual install of the Kubeflow Pipelines, make sure that the
+* The path for your bucket should be in this format `s3:\\your-bucket`.
+{% endtab %}
 
 {% tab title="GCP" %}
-
-- [Create](https://cloud.google.com/storage/docs/creating-buckets) a GCP Cloud Storage Bucket in a region of your choice.
-- Make sure that your GKE cluster is authorized to access the GCS bucket.
-- The path for your bucket should be in this format `gs://your-bucket`
-
+* [Create](https://cloud.google.com/storage/docs/creating-buckets) a GCP Cloud Storage Bucket in a region of your choice.
+* Make sure that your GKE cluster is authorized to access the GCS bucket.
+* The path for your bucket should be in this format `gs://your-bucket`.
 {% endtab %}
 
-
-{% tab title="Azure" %} 
-
-- [Create](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) an Azure Storage Account and [add](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal) a *container* to hold your *blobs.*
-- Open your storage account page on the Azure portal and follow these steps:
-    - Keep the name of the storage account handy - it will be required in a later step.
-    - From the navigation panel on the left, select **Access Keys** under **Security + networking**. Note any of the keys available there for future use.
-- Add your storage `Account Name` and the `Account Key` you just created as environment variables in your Docker image. This can be done by extending the `zenmldocker/zenml` image and [adding](https://docs.docker.com/engine/reference/builder/#env) these variables in a Dockerfile. 
-This image can then be used by modifying the `custom_docker_base_image_name` field inside your orchestrator’s YAML configuration at `.zen/orchestrators/*.yaml` .
-- The path for your bucket should be in this format `az://<CONTAINER-NAME>`
-
-{% endtab %} 
-
+{% tab title="Azure" %}
+* [Create](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) an Azure Storage Account and [add](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal) a _container_ to hold your _blobs._
+* Open your storage account page on the Azure portal and follow these steps:
+  * Keep the name of the storage account handy - it will be required in a later step.
+  * From the navigation panel on the left, select **Access Keys** under **Security + networking**. Note any of the keys available there for future use.
+* Make sure to set a combination of the following environment variables:`AZURE_STORAGE_CONNECTION_STRING` or `AZURE_STORAGE_ACCOUNT_NAME` and one of \[`AZURE_STORAGE_ACCOUNT_KEY`, `AZURE_STORAGE_SAS_TOKEN`]. \
+  This can be done by extending the `zenmldocker/zenml` image and [adding](https://docs.docker.com/engine/reference/builder/#env) these variables in a Dockerfile. This image can then be used by modifying the `custom_docker_base_image_name` field inside your orchestrator’s YAML configuration at `.zen/orchestrators/*.yaml` .
```

Comment:
Maybe we can leave this "hack" out and specify the base image when creating the orchestrator later?
```zenml orchestrator register kubeflow_orchestrator --type=kubeflow --custom_docker_base_image_name=...```

Reply:
I agree - much cleaner.